domehunter/dome_control.py

```diff
@@ -44,6 +46,32 @@ class DomeCommandError(Exception):  # pragma: no cover
     pass
 
 
+def load_dome_config(config_path=None):
+    """Load dome configuration infomation from a yaml file.
+
+    Parameters
+    ----------
+    config_path : str
+        File path of desired configuration yaml file.
+
+    Returns
+    -------
+    type dict
+        Dictionary of keyword args to pass through to dome instance.
+
+    """
+    if config_path is None:
+        directory = os.path.abspath(os.path.dirname(__file__))
+        rel_path = 'gRPC-server/dome_controller_config.yml'
+        config_path = os.path.join(directory, rel_path)
+    try:
+        with open(config_path, 'rb') as f:
```

Comment:
I don't think you want to read this as bytes.

```suggestion
        with open(config_path, 'r') as f:
```